src/Take.Blip.Builder/AnalyzeBuilderExceptions.cs

```diff
@@ -18,6 +18,7 @@ public AnalyzeBuilderExceptions()
             _validations = new List<Func<Exception, bool>>()
             {
                 ex => ex is ActionProcessingException && ex?.InnerException is ArgumentException,
+                ex => ex is ActionProcessingException && ex?.InnerException is NullReferenceException,
```

Comment:
I'm afraid of been too general using "NullReferenceException". 
is there no other more specific validation?